src/app/shared/analytics.service.ts

```diff
@@ -16,6 +16,26 @@ declare global {
 
 @Injectable()
 export class AnalyticService {
+  private eventKeyMap = {
+    'showAddAppOverlay': 'add app opened',
+    'closeAddAppOverlay': 'add app closed',
+    'showCreateApp': 'create app opened',
+    'closeCreateApp': 'create app closed',
+    'showImportApp': 'import app opened',
+    'closeImportApp': 'import app closed',
+    'clickContinueAppOverlay': 'click continue app overlay',
+    'completeMissionRuntimeStep': 'mission runtime completed',
+    'completeDependencyEditorStep': 'dependency editor completed',
+    'completePipelineStep_Create': 'pipeline completed in create',
+    'completeGitProviderStep_Create': 'git provider completed in create',
+    'completeSummaryStep_Create': 'summary completed in create',
+    'completePipelineStep_Import': 'pipeline completed in import',
+    'completeGitProviderStep_Import': 'git provider completed in import',
+    'completeSummaryStep_Import': 'summary completed in import',
+    'viewApplicationButtonClicked': 'view application button clicked',
+    'stepIndicatorClicked': 'step indicator clicked',
+    'stepIndicatorProjectInputClicked': 'step indicator project name clicked'
```

Comment:
Can't this mapping be on the `ngx-launcher` side? This way you still couple both code bases.